tensorboardX/graph.py

```diff
@@ -48,11 +48,13 @@ def parse(graph):
 
 def graph(model, args, verbose=False):
     import torch
-    with torch.onnx.set_training(model, False):
-        trace, _ = torch.jit.get_trace_graph(model, args)
     if LooseVersion(torch.__version__) >= LooseVersion("0.4"):
```

Comment:
You're right, it seems like they just renamed the function legacy_trace -> create_trace -> get_trace_graph in a short time frame. It would be better to avoid more comprehensive version control criterion. I'll have a look into this.

Reply:
Based on the 90 days release cycle. I guess pre-build v0.4 will be released soon. We should decide the criterion after then.

Reply:
There is a `torch-nightly` conda package, with the corresponding to the day on which the binary was built. In my case, `python -c "import torch; print(torch.__version__)"` gives `2018.04.10`.

Maybe the criterion could be based on this for the time being?

Reply:
@ostrokach That makes sense. In fact, the CI is tested against pytorch-nightly. Should be a simple PR.